RELEASE_NOTES.md

```diff
@@ -1,5 +1,80 @@
 # Release Notes
 
+## stable-1.15-2.3.0, stable-1.16-2.3.0, stable-1.17-2.3.0
+
+- Azuredisk-csi-driver: 
+  - enable the Snapshot controller ([#443](https://github.com/mesosphere/kubernetes-base-addons/pull/443), [@dkoshkin](https://github.com/dkoshkin))
+- Cert-manager:
+  - `Issuer` namespace setable
+    - `Certificate` namespace setable ([#378](https://github.com/mesosphere/kubernetes-base-addons/pull/378), [@sebbrandt87](https://github.com/sebbrandt87))
+- Dex-k8s-authenticator:
+  - Windows download support for the credentials plugin ([#377](https://github.com/mesosphere/kubernetes-base-addons/pull/377), [@jr0d](https://github.com/jr0d))
+  - Fixed bug causing `certificate-authority=`  option to be added to token instructions on the windows tab when it should have been omitted. ([#436](https://github.com/mesosphere/kubernetes-base-addons/pull/436), [@jr0d](https://github.com/jr0d))
+- Elasticsearch-curator:
+  - version 5.8.1 ([#374](https://github.com/mesosphere/kubernetes-base-addons/pull/374), [@sebbrandt87](https://github.com/sebbrandt87))
+  - Added value `cronjob.startingDeadlineSeconds`: Amount of time to try reschedule job if we can't run on time ([#447](https://github.com/mesosphere/kubernetes-base-addons/pull/447), [@d2iq-dispatch](https://github.com/d2iq-dispatch))
+- Elasticsearch-exporter: 
+  - updated from 2.11 to 3.7.0
+    - Add a parameter for the elasticsearch-exporter: es.indices_settings as it is supported since version 1.0.4 (the elasticsearch-exporter chart is supporting the version 1.1.0)
+    - Update description for envFromSecret parameter in readme
+    - Feature flap the flag es.uri to allow fallback to env var ES_URI
+    - Allow setting environment variables with k8s secret information to support referencing already existing sensitive parameters.
+    - Add es.ssl.client.enabled value for better functionality readability
+    - Add option to disable client cert auth in Elasticsearch exporter
+    - Add the serviceMonitor targetLabels key as documented in the Prometheus Operator API
+    - Add log.level and log.format configs
+    - Add the ServiceMonitor metricRelabelings key as documented in the Prometheus Operator API
+    - Add sampleLimit configuration option ([#449](https://github.com/mesosphere/kubernetes-base-addons/pull/449), [@d2iq-dispatch](https://github.com/d2iq-dispatch))
+- Fluent-bit:
+  - Three different elasticsearch indicies created
+    - kubernetes_cluster-- (for container logs)
+    - kubernetes_audit-- (for audit logs from kube-apiserver)
+    - kubernetes_host-- (for all systemd host logs)
+  - Pods no longer run priviledged ([#371](https://github.com/mesosphere/kubernetes-base-addons/pull/371), [@sebbrandt87](https://github.com/sebbrandt87))
```

Comment:
should we may remove this line? as within the next PR we fix this to be run  `priviledged` again as we otherwise cannot access `/dev/kmsg` and other logs.